maxim/tests/test_livekit.py

```
@@ -0,0 +1,49 @@
+        room=ctx.room,
+        agent=Assistant(),
+        room_input_options=RoomInputOptions(
+            # LiveKit Cloud enhanced noise cancellation
+            # - If self-hosting, omit this parameter
+            # - For telephony applications, use `BVCTelephony` for best results
+            noise_cancellation=noise_cancellation.BVC(),
+        ),
+    )
+
+    await session.generate_reply(
+        instructions="Greet the user and offer your assistance."
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)